doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,139 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run `VxWorks`_ as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
+NUC (model NUC7i5DNHE) in this tutorial. Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to VxWorks
+***********************
+
+VxWorks®, the industry’s leading real-time operating system, delivers all the performance, determinism, reliability,
+safety, and security capabilities you need to meet the highest standards for running the computing systems of the most
+important critical infrastructure.
+
+Steps for Using VxWorks as User OS
+**********************************
+
+#. Build VxWorks
+
+   Follow the `VxWorks Getting Started Guide <https://docs.windriver.com/bundle/vxworks_7_tutorial_kernel_application_workbench_sr0610/page/rbu1422461642318.html>`_
```

Comment:
I get an error when I follow this link: You are not permitted to view this content 
or you need to login in order to access.  Do you need to buy VxWorks (and have an account) to do this tutorial?

Reply:
Yeah. We need buy it. I will add the note you suggested above.